server/chat-commands/info.ts

```diff
@@ -491,6 +491,11 @@ export const commands: ChatCommands = {
 				Utils.html`${curUser.connected ? ` \u25C9 ` : ` \u25CC `} ${curUser.name}`
 			);
 		}
+		results.sort((aName, bName) => {
+			if (aName.includes(`\u25CC`) && bName.includes(`\u25C9`)) return 1;
+			return -1;
+		});
```

Comment:
This is not an ordering and can't be used in a sort. :/

It always returns `-1` if both users are online (or both users are offline), which is incorrectly ordered (they can't both be after the other). You should return `0` if both are online or both are offline:

```ts
		results.sort((aName, bName) => (
			(aName.includes(`\u25CC` ? 1 : 0) - (aName.includes(`\u25CC` ? 1 : 0)
		);
```

All of this is, of course, entirely unnecessary because I wrote a sort helper in Utils so you'd never have to deal with that again.

```suggestion
		Utils.sortBy(results, name => name.startsWith('  \u25C9'));
```